packages/cli/src/License.ts

```diff
@@ -357,6 +357,12 @@ export class License {
 		return this.getFeatureValue(LICENSE_QUOTAS.TEAM_PROJECT_LIMIT) ?? 0;
 	}
 
+	getConcurrencyProductionCap() {
+		return (
+			this.getFeatureValue(LICENSE_QUOTAS.CONCURRENCY_PRODUCTION_CAP) ?? UNLIMITED_LICENSE_QUOTA
```

Comment:
have we discussed that we want to use the license to determine concurrency limits? if yes, do we still want to use the `executions.concurrency.productionCap` config variable?
It feels like one or the other should suffice.

Reply:
From [requirements](https://www.notion.so/n8n/Concurrency-limits-for-main-mode-8b642df5e98a479cadd1cfc4950d27a1):

![Capture 2024-06-06 at 11 10 38@2x](https://github.com/n8n-io/n8n/assets/44588767/74beab18-6e51-4607-be36-6e00fe9c4e90)

Reply:
So from reading this, I can only see mentions to creating environment variables, nothing license related.
For cloud where we want to impose some limits (to safeguard user instances) this can be done via environment variables, there's no need for any sort of licensing here, right?

Reply:
We could impose those limits using the ephemeral licenses but that's not necessary, we can simply use environment variables for this, unless we really want to centralize it all with licenses, but it seems to me we're mixing requirements

Reply:
See [2351580](https://github.com/n8n-io/n8n/pull/9453/commits/235158063a1e38a9377048f38067df62004af45d)